addon/controllers/project/form.js

```diff
@@ -22,6 +22,10 @@ export default class ProjectFormController extends ImportController {
   @service intl;
   @service notification;
 
+  INFRASTRUCTURE = 6010;
+  SUPERSTRUCTURE = 6011;
+  SPECIALSTRUCTURE = 6012;
+
```

Comment:
We defined most of  the options in the models, should we keep it the same?

Reply:
Done